src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -400,7 +419,14 @@ private PolicyModel deletePolicyMemberInner(AuthenticatedUserRequest userReq,
             resourceId.toString(),
             policyName,
             userEmail);
+        return null;
```

Comment:
Any reason we're returning `Void` and not just `void` (so we wouldn't need the return statements at all)

Reply:
The SamRetry.perform() method requires that you return something of type variable T. So, setting the return type to Void and returning null satisfies this requirement.